python/requirements.txt

```diff
@@ -1,4 +1,4 @@
 six
 numpy==1.15.4; python_version == "3.4"
-numpy; python_version != "3.4"
+numpy==1.16.4; python_version != "3.4"
```

Comment:
I think we should we hard-code only for Python 2.7 and 3.4.
Let the user use latest version (or version required by other packages) in other cases.

Reply:
Done